hw/opentitan/ot_spi_device.c

```diff
@@ -713,6 +762,16 @@ static void ot_spi_device_clear_modes(OtSPIDeviceState *s)
     f->payload += SPI_SRAM_INGRESS_OFFSET;
     memset(f->buffer, 0u, SPI_FLASH_BUFFER_SIZE);
 
+    SpiDeviceTpm *tpm = &s->tpm;
+    tpm->state = 0;
+    tpm->can_receive = 1;
+    tpm->transfer_size = 0;
+    tpm->read = 0;
```

Comment:
Nit - prefer bool constant:

```suggestion
    tpm->read = false;
```